README.md

```diff
@@ -76,6 +78,30 @@ For the above example, we implemented the !roll Lego.. Now, to see this in actio
 <parsec> !roll d20
 <legobot> You Rolled: 2
 ```
+
+## Development
+
+**Information will be added as becomes relevant**
+
+When developing and running the development server or installing locally, you must install the requirment. To run the development server, just use vagrant! First, make sure you are in the Legobot directory. Then:
+
+```
+sudo $YourDistroPackageManager $install vagrant

```

Comment:
Add a link to Vagrant's install guide instead. It will prevent us from having to keep up with their install docs. It's like a symlink, but for knowledge!